spec/turn_spec.rb

```diff
@@ -0,0 +1,202 @@
+    turn.pile_cards
+    expect(turn.spoils_of_war).to eq([@card1, @card2, @card5, @card4, @card3, @card6])
+  end
+
+  it 'war award_spoils' do
+    deck1 = Deck.new([@card1, @card2, @card5, @card8])
+    deck2 = Deck.new([@card4, @card3, @card6, @card7])
+    player1 = Player.new("Megan", deck1)
+    player2 = Player.new("Aurora", deck2)
+    turn = Turn.new(player1, player2)
+    winner = turn.winner
+
+    turn.pile_cards
+    expect(turn.spoils_of_war).to eq([@card1, @card2, @card5, @card4, @card3, @card6])
+    turn.award_spoils(player2)
+    expect(winner.deck.cards).to eq([@card7, @card1, @card2, @card5, @card4, @card3, @card6])
+  end
+
+  it 'has mutually assured destruction' do
+    @card1 = Card.new(:heart, 'Jack', 11)
+    @card2 = Card.new(:heart, '10', 10)
+    @card3 = Card.new(:heart, '9', 9)
+    @card4 = Card.new(:diamond, 'Jack', 11)
+    @card5 = Card.new(:heart, '8', 8)
+    @card6 = Card.new(:diamond, '8', 8)
+    @card7 = Card.new(:heart, '3', 3)
+    @card8 = Card.new(:diamond, '2', 2)
+    deck1 = Deck.new([@card1, @card2, @card5, @card8])
+    deck2 = Deck.new([@card4, @card3, @card6, @card7])
+    player1 = Player.new("Megan", deck1)
+    player2 = Player.new("Aurora", deck2)
+    turn = Turn.new(player1, player2)
+    expect(turn.type).to eq(:mutually_assured_destruction)
+  end
+
+  it 'has mutal winner' do
+    @card1 = Card.new(:heart, 'Jack', 11)
+    @card2 = Card.new(:heart, '10', 10)
+    @card3 = Card.new(:heart, '9', 9)
+    @card4 = Card.new(:diamond, 'Jack', 11)
+    @card5 = Card.new(:heart, '8', 8)
+    @card6 = Card.new(:diamond, '8', 8)
+    @card7 = Card.new(:heart, '3', 3)
+    @card8 = Card.new(:diamond, '2', 2)
+    deck1 = Deck.new([@card1, @card2, @card5, @card8])
+    deck2 = Deck.new([@card4, @card3, @card6, @card7])
+    player1 = Player.new("Megan", deck1)
+    player2 = Player.new("Aurora", deck2)
+    turn = Turn.new(player1, player2)
+    winner = turn.winner
+  expect(turn.type).to eq(:mutually_assured_destruction)
+  expect(turn.winner).to eq("No Winner")
+  end
+
+  it 'pile cards mutual' do
+    @card1 = Card.new(:heart, 'Jack', 11)
+    @card2 = Card.new(:heart, '10', 10)
+    @card3 = Card.new(:heart, '9', 9)
+    @card4 = Card.new(:diamond, 'Jack', 11)
+    @card5 = Card.new(:heart, '8', 8)
+    @card6 = Card.new(:diamond, '8', 8)
+    @card7 = Card.new(:heart, '3', 3)
+    @card8 = Card.new(:diamond, '2', 2)
+    deck1 = Deck.new([@card1, @card2, @card5, @card8])
+    deck2 = Deck.new([@card4, @card3, @card6, @card7])
+    player1 = Player.new("Megan", deck1)
+    player2 = Player.new("Aurora", deck2)
+    turn = Turn.new(player1, player2)
+    winner = turn.winner
+    expect(turn.type).to eq(:mutually_assured_destruction)
+    expect(player1.deck.cards.size).to eq(4)
+    expect(player2.deck.cards.size).to eq(4)
+    turn.spoils_of_war
+    turn.pile_cards
+    expect(player1.deck.cards.size).to eq(1)
+    expect(player2.deck.cards.size).to eq(1)
+  end
+end
```

Comment:
The organization of the tests in this file was a little hard to follow, it jumped around quite a bit. It would have probably been better to test all the `:basic` scenarios first, then test all the `:war` scenarios, and then the `:mutually_assured_destruction` scenarios after.

You are also missing an `award_spoils` test for `:mutually_assured_destrcution`. I believe had you included this test you would have seen that there was potential for the code to break in your `award_spoils` method if it were passed `"No Winner"`.